hack/generator/pkg/astmodel/package_definition.go

```diff
@@ -234,7 +242,8 @@ func emitGroupVersionFile(pkgDef *PackageDefinition, outputDir string) error {
 		return err
 	}
 
-	gvFile := filepath.Join(outputDir, "groupversion_info.go")
+	// TODO: The name of this file doesn't matter, right?
```

Comment:
I dont think that it does give that controller-gen seems happy with it named with a prefix still, although we actually don't need the `zz_generated` prefix - do we want it or no?

Reply:
See my other comment. `zz_generated` should be bathed in high velocity hot plasma:

![image](https://user-images.githubusercontent.com/1272094/83981084-ddbeff00-a96e-11ea-880b-93ab2f301213.png)

Reply:
I definitely don't think we should copy that naming scheme. For one, I think it could confuse people into thinking those files had been generated by Kubebuilder. Secondly, it's pretty ghastly.